src/emulation/mod.rs

```diff
@@ -76,6 +76,24 @@ impl SyscallMock {
                     ptrace::setregs(pid, registers)?;
                 }
             }
+            (Syscall::Stat, SyscallStop::Exit) if !self.protocol.mocked_files.is_empty() => {
+                let filename = tracee_memory::peek_string(pid, registers.rdi)?;
+                if self.protocol.mocked_files.contains(&filename) {
+                    let buffer_ptr = registers.rsi;
+                    #[allow(clippy::forget_copy)]
```

Comment:
i'm not sure if this is ok... but clippy is complaining about something that happens after the macro expands :'(

Reply:
Yeah, I think, this is fine. Should probably be fixed in `memoffset`, you could create an issue there.